db/table_cache.cc

```diff
@@ -171,7 +171,8 @@ Status TableCache::FindTable(const ReadOptions& ro,
 
   if (*handle == nullptr) {
     if (no_io) {  // Don't do IO and return a not-found status
-      return Status::Incomplete("Table not found in table_cache, no_io is set");
+      s = Status::Incomplete("Table not found in table_cache, no_io is set");
```

Comment:
Please fix the comment on the earlier line

Reply:
Deleted the comment since it was just inaccurately repeating the code.